### ISSUE TITLE
Turning ON/OFF the VPN first time after signing up is not possible

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-services/guardian-vpn-windows#144]
#### Affected versions:
- 1.2 (latest received version);

#### Affected Platforms:
- Windows 10 x64;

#### Prerequisites:
- Have the VPN app installed;

#### Steps to reproduce:
1. Open the VPN app;
2. Click on the “Get Started” button;
3. Sign in using a new FxA account without subscription;
4. Finalize sign in and subscription;
5. Sign out;
6. Open the VPN and sign back in using the same account;
7. Turn on the VPN;

#### Expected result:
- The VPN is turned on with succes.

#### Actual result:
- The VPN is not turned on. The VPN remains in a “Connecting” state.

Notes:
- this also happens if at step 6 we use a different, old account;
- same happens if we want to turn off the VPN;
- if we “exit” the app and reopen it, the issue gets fixed;
![connecting](https://user-images.githubusercontent.com/15685960/97697905-5bb3ac00-1ab0-11eb-9cc3-4243f4a2b5ef.gif)

- We’ve tried to reproduce the new issue on the old build (the one before this) and we couldn’t. After step 7 (check the issue) a red banner is prompted, informing the user that he must restart the client.
![unable](https://user-images.githubusercontent.com/15685960/97710081-f2d62f00-1ac3-11eb-814f-feb04105dae9.png)

**Comment by chrismore** [comment]
@ValentinaPC between steps 6 and 7, can you try the steps to reproduce again and wait about a minute between steps 6 and 7 and see if you can still reproduce this?

**Comment by ValentinaPC** [comment]
@chrismore Yeah, tried that before logging the issue. Same results.
No matter how long you wait, or what account you use after, or how many times you try it. If you first use a fresh-account and then sign out...you will be able to reproduce it, each time. Only the uninstall or exit will fix it.

**Comment by ValentinaPC** [comment]
Verified this as fixed on the latest VPN received build, using Windows 10 x64.
VPN app can be turned on after making a subscription and signing back in.
